### PR TITLE
Remove unnecessary code

### PR DESCRIPTION
**Pull request by murdos**
Just a little cleanup in the continuation of https://github.com/jhipster/jhipster-kotlin/pull/146 and https://github.com/jhipster/generator-jhipster/pull/9942

### CODE REVIEW

File: generators/entity-server/index.js [hunk 1]
```diff
@@ -24,14 +24,9 @@ const writeFiles = require('./files').writeFiles;
 module.exports = class extends EntityServerGenerator {
     constructor(args, opts) {
         super(args, { fromBlueprint: true, ...opts }); // fromBlueprint variable is important
-
-        const jhContext = (this.jhipsterContext = this.options.jhipsterContext || opts.jhipsterContext);
```
Comment (sendilkumarn) [inline generators/entity-server/index.js]: I think `this.jhipsterContext` assignment is valid and we need to have it.

Comment (murdos) [reply generators/entity-server/index.js]: According to https://github.com/jhipster/generator-jhipster/pull/9942 it's done by parent generator, so you don't need to do it in blueprint generator

Comment (sendilkumarn) [reply generators/entity-server/index.js]: Interesting, let me have a look. Sorry for taking too long to reply. Catching up things on the opensource side.

Comment (sendilkumarn) [reply generators/entity-server/index.js]: That is interesting.

**Comment by sendilkumarn** [comment]
Thanks @murdos for the pull request 👍

**Pull request merged**